clients/cobol-lsp-vscode-extension/src/web/test/suite/extension.test.ts

```diff
@@ -0,0 +1,15 @@
+import * as assert from 'assert';
+
+// You can import and use all API from the 'vscode' module
+// as well as import your extension to test it
+import * as vscode from 'vscode';
+// import * as myExtension from '../../extension';
+
+suite('Web Extension Test Suite', () => {
+	vscode.window.showInformationMessage('Start all tests.');
```

Comment:
Where and how is this test run in the CI?

Reply:
The tests for a web extension require it to be run in a web browser, according to [this documentation](https://code.visualstudio.com/api/extension-guides/web-extensions#web-extension-tests) from Microsoft.

Currently the unit test in the commit is only the one that was included in the template generation code given by Microsoft.

Reply:
For testing in CI, https://github.com/microsoft/vscode-test-web package can be used.